content/en/apps/guides/forum-offline.md

```diff
@@ -1,14 +1,14 @@
 ## Overview
 
-In this post we look at setting up an instance of the [CHT server](https://communityhealthtoolkit.org/) that is initially online so it can be configured, but ultimately is deployed fully offline.  We'll be using the information from the newly published [Offline CHT Server](https://docs.communityhealthtoolkit.org/apps/guides/offline/) guide in the CHT docs as well as the existing guide on [how to deploy a self-hosted CHT server](https://github.com/medic/cht-infrastructure/tree/master/self-hosting).
+In this post we look at setting up an instance of the [CHT server](https://communityhealthtoolkit.org/) that is initially online so it can be configured, but ultimately is deployed fully offline.  We'll be using the information from the newly published [Offline CHT Server](hhttps://docs.communityhealthtoolkit.org/apps/guides/hosting/offline/) guide in the CHT docs as well as the existing guide on [how to deploy a self-hosted CHT server](https://docs.communityhealthtoolkit.org/apps/guides/hosting/self-hosting/).
```

Comment:
`[Offline CHT Server](hhttps://docs.communityhealthtoolkit.org/apps/guides/hosting/offline/)`

There's an extra `h` in  `hhttps` :-)